shm/Cargo.cpp

```diff
@@ -0,0 +1,31 @@
+#include "Cargo.hpp"
+
+Cargo& Cargo::operator+=(size_t amount)
+{
+    amount_ += amount;
+    return *this;
+}
+Cargo& Cargo::operator-=(size_t amount)
+{
+    amount_ -= amount;
+    return *this;
```

Comment:
How are you handling negative cargo ammount?

Reply:
+1